jupyter_server/services/kernels/kernelmanager.py

```diff
@@ -12,24 +12,24 @@
 from functools import partial
 import os
 
-from tornado import gen, web
+from tornado import web
 from tornado.concurrent import Future
 from tornado.ioloop import IOLoop, PeriodicCallback
 
 from jupyter_client.session import Session
-from jupyter_client.multikernelmanager import MultiKernelManager
+from jupyter_client.multikernelmanager import AsyncMultiKernelManager
 from traitlets import (Any, Bool, Dict, List, Unicode, TraitError, Integer,
        Float, Instance, default, validate
 )
 
-from jupyter_server.utils import maybe_future, to_os_path, exists
+from jupyter_server.utils import to_os_path, exists, call_blocking
 from jupyter_server._tz import utcnow, isoformat
 from ipython_genutils.py3compat import getcwd
 
 from jupyter_server.prometheus.metrics import KERNEL_CURRENTLY_RUNNING_TOTAL
 
 
-class MappingKernelManager(MultiKernelManager):
+class MappingKernelManager(AsyncMultiKernelManager):
```

Comment:
I'm wondering if we should rename this to `AsyncMappingKernelManager` to avoid confusion when working between notebook and jupyter_server repos.  I'm hoping that https://github.com/jupyter/notebook/pull/4479, which introduces _optional_  async management,  will be merged soon.

Reply:
You're right, I will rename it.